src/Slawek/Fsd.Slawek.Cs.Ex3/Fsd.Slawek.Cs.Ex3/Models/UserService.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+
+namespace Fsd.Slawek.Cs.Ex3.Models
+{
+    public class UserService
```

Comment:
troche pomieszane odpowiedzialności. user service który zachowuje się jak prezenter? jakoś mi się to nie klei w całość